examples/sn-blog-gatsby/package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "build": "gatsby build",
     "clean": "rimraf public",
```

Comment:
use gatsby clean instead: "clean": "gatsby clean"
and rimraf is removable from dependencies